rainbows.rb

```diff
@@ -17,6 +17,10 @@
   [255,0,125] => "raspberry"
 }
 
+rainbow = [{:red => {r: 255, g: 0, b: 0}}, {:orange => {r: 255, g: 125, b: 0}}, 
+  {:yellow => {r: 255, g: 255, b: 0}}, {:green => {r: 0, g: 255, b: 0}}, {:blue => {r: 0, g: 0, b: 255}}, 
+  {:indigo => {r: 75, g: 0, b: 130}}, {:violet => {r: 125, g: 0, b: 255}}]
+
```

Comment:
You should separate these onto separate line for readability and ease in removing/commenting out lines.